src/main/java/owlmoney/logic/parser/ParseCommand.java

```diff
@@ -0,0 +1,88 @@
+package owlmoney.logic.parser;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+import owlmoney.logic.command.Command;
+import owlmoney.logic.command.ExitCommand;
+import owlmoney.logic.parser.exception.ParserException;
+
+/**
+ * Represents the first instance of parsing user input.
+ * This determines what type of command the user has entered.
+ */
+
+public class ParseCommand extends Parser {
+    private ParseType parseType = new ParseType();
+    private final Scanner scanner = new Scanner(System.in);
+    private static final String[] COMMAND_KEYWORDS = new String[] {
```

Comment:
```
Hi @jylee-git ,

With regards to your:

The contents of your COMMAND_KEYWORDS are equivalent to magic numbers.

I thought we did the right thing by putting all the supposedly magic strings in a declared constant with private static final?

Regards
Shamus Neo
```

Maybe let me rephrase it: It's equivalent to dumping a collection of magic numbers inside.

Because just dumping these `"/help", "/undo", "/add", "/edit", "/delete", "/list", "/find", "/transfer", "/exit"` don't explain what are these values about.

So for this sort of case, you should declare static final variables for each these commands, then dump them in to your `private static final List<String> COMMAND_KEYWORD_LISTS`.

Reply:
Alright noted, will correct them